avm/res/resource-graph/query/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,50 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = '''
+This instance deploys the module with the minimum set of required parameters.
+> **Note:** The test currently implements additional non-required parameters to cater for a test-specific limitation.
+'''
```

```suggestion
metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```

Actually you don't ;) Requires an update of the readme.md